src/3rd_party-static/MessageBroker/src/client/mb_controller.cpp

```diff
@@ -6,347 +6,308 @@
 
 #include "mb_controller.hpp"
 
-#include "MBDebugHelper.h" 
+#include "MBDebugHelper.h"
 #include "CMessageBroker.hpp"
 
-namespace NsMessageBroker
-{
-   CMessageBrokerController::CMessageBrokerController(const std::string& address, uint16_t port, std::string name):
-   TcpClient(address, port),
-   stop(false),
-   m_receivingBuffer(""),
-   mControllersIdStart(-1),
-   mControllersIdCurrent(0)
-   {
-      mControllersName = name;
-   }
+namespace NsMessageBroker {
+CMessageBrokerController::CMessageBrokerController(const std::string& address,
+                                                   uint16_t port,
+                                                   std::string name)
+    : TcpClient(address, port)
+    , stop(false)
+    , m_receivingBuffer("")
+    , mControllersIdStart(-1)
+    , mControllersIdCurrent(0) {
+  mControllersName = name;
+}
 
-   std::string CMessageBrokerController::getControllersName()
-   {
-      return mControllersName;
-   }
+std::string CMessageBrokerController::getControllersName() {
```

Comment:
@AByzhynar may be but I am rely not sure, and I wont do it now because of time restrictions.